Makefile

```diff
@@ -69,14 +69,11 @@ test-all-components-coverage: ## Run tests with coverage
 		$(MAKE) test-components-coverage GROUP=$$(basename $$component_group) ; \
 	done
 
-run: ## Compile pipeline and run pipeline in sandbox environment. Must specify pipeline=<training|prediction>. Optionally specify ENABLE_PIPELINE_CACHING=<true|false> (defaults to default Vertex caching behaviour)
+wait ?= true
```

Comment:
change default to `false` - when using it locally we want to return immediately, when using it in CI we will set `wait=true`